planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -712,6 +728,28 @@ void GoalPlannerModule::setStopPath(BehaviorModuleOutput & output)
   }
 }
 
+void GoalPlannerModule::setStopPathInCurrentPath(BehaviorModuleOutput & output)
+{
+  if (status_.prev_is_safe_dynamic_objects || status_.prev_stop_path_after_approval == nullptr) {
+    // safe -> not_safe or no prev_stop_path: generate new stop_path
+    output.path = std::make_shared<PathWithLaneId>(generateStopPointInCurrentPath());
+    output.reference_path = getPreviousModuleOutput().reference_path;
+    status_.prev_stop_path_after_approval = output.path;
+    // set stop path as pull over path
+    mutex_.lock();
+    last_path_update_time_ = std::make_unique<rclcpp::Time>(clock_->now());
+    mutex_.unlock();
+    RCLCPP_WARN_THROTTLE(
+      getLogger(), *clock_, 5000, "Found approved pull_over path is not safe, generate stop path");
+  } else {
+    // not_safe -> not_safe: use previous stop path
+    output.path = status_.prev_stop_path_after_approval;
+    output.reference_path = getPreviousModuleOutput().reference_path;
+    RCLCPP_WARN_THROTTLE(
+      getLogger(), *clock_, 5000, "Found approved pull_over path is not safe, generate stop path");
+  }
+}
+
```

Comment:
move same processing out of if condition

```suggestion
  if (status_.prev_is_safe_dynamic_objects || status_.prev_stop_path_after_approval == nullptr) {
    // safe -> not_safe or no prev_stop_path: generate new stop_path
    output.path = std::make_shared<PathWithLaneId>(generateStopPointInCurrentPath());
    status_.prev_stop_path_after_approval = output.path;
    // set stop path as pull over path
    mutex_.lock();
    last_path_update_time_ = std::make_unique<rclcpp::Time>(clock_->now());
    mutex_.unlock();
  } else {
    // not_safe -> not_safe: use previous stop path
    output.path = status_.prev_stop_path_after_approval;
  }
  output.reference_path = getPreviousModuleOutput().reference_path;
   RCLCPP_WARN_THROTTLE(
      getLogger(), *clock_, 5000, "Found approved pull_over path is not safe, generate stop path");  
}

```

Reply:
ops thank you!
I forgot to change here!

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/4927/commits/938ac7910189c1cbd2cf1d122f415b6f94f251f5